### DISCUSSION TITLE
customizing the output format of the stdout output plugin

### DISCUSSION THREAD

**Discussion by BayanAzima** [fluent/fluent-bit#8693] [General]
Hi there,

I am trying to print out logs after being parsed in a custom format with the stdout output plugin. According to the docs there are 3 formats (msgpack json, json_lines and json_stream.) but no way to provide a custom template.

What i want to do is similar to the template field in the File plugin (https://docs.fluentbit.io/manual/pipeline/outputs/file#template-format)

Is there a way to customize the format of the stdout plugin or another way to achieve the same goal?

**Comment by patrick-stephens** [comment]
I don't believe so.
Maybe just use another filter first to mangle the output however you want?
Also stdout is "just" a file (on Linux) so you could technically use the file plugin for it if you felt particularly disgusting...
